ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -1,6 +1,7 @@
 import ivy
 from ivy.functional.frontends.jax.func_wrapper import to_ivy_arrays_and_back
 from ivy.func_wrapper import with_unsupported_dtypes
+import numpy as np_frontend
```

Comment:
We're not supposed to use native packages like `jax`, `numpy`, `torch` etc. in the frontends. Instead, we only implement the frontend functions using either `ivy` functions or functions from the same frontend e.g. `jax` frontend